pkg/minishift/profile/profile.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package profile
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/util/filehelper"
+)
+
+// Returns the list of profile names
+func GetProfileList() []string {
+	var profileList []string
+	baseDir := constants.GetMinishiftHomeDir()
+	profileBaseDir := filepath.Join(baseDir, "profiles")
+
+	if !filehelper.IsDirectory(baseDir) {
+		return profileList
+	} else {
```

Comment:
We can write `profileList = append(profileList, constants.DefaultProfileName)` after `if` since if control goes into if block it is returning from function.

Reply:
in short, he means "remove `else` since it obscures the intent"